### ISSUE TITLE
Generate local svg with white background

### ISSUE THREAD

**Issue by rcp1** [TeamMeow/vscode-math-to-image#4]
Hi,

first, congrats for this amazing tool! :)

Since I work with dark themed VSC but the browser setting is user dependent, it would be nice to add the option to allow a white background in the locally saved SVG to always have contrast for light and dark setting of browser or VSC.

Best

**Comment by devel0** [comment]
I had the same problem, as a workaround you can set follow in your project `.vscode/settings.json`

```json
{
  "markdown.styles": ["markdown-styles.css"]
}
```

then create a css file in the root workspace project `markdown-styles.css` with this content

```css
img {	
    background-color: white;
}
```

for a result like follow

![image](https://user-images.githubusercontent.com/13405008/101864609-89424980-3b74-11eb-9325-955f2ec220f0.png)

from markdown containing svg eq

```md
<!-- $$
accel(x)=\frac{2\cdot t}{d}\cdot\left(1-\cos\left(\frac{4\cdot\pi\cdot x}{d}\right)\right)
$$ --> 

<div align="center"><img src="svg/3HfEZfsh6r.svg"/></div>
```

**Comment by spencerwooo** [comment]
Feature implemented as of version v0.1.0, closing.

**Comment by rcp1** [comment]
Awesome, thanks!
